sql/core/src/main/scala/org/apache/spark/sql/execution/command/DataWritingCommand.scala

```diff
@@ -96,4 +98,20 @@ object DataWritingCommand {
       sparkContext.getLocalProperty(SQLExecution.EXECUTION_ID_KEY),
       metrics.values.toSeq)
   }
+
+  def assertEmptyRootPath(locationUri: Option[URI], configuration: Configuration) {
+    locationUri.fold() { tablePathUri: URI =>
+      val tablePath = tablePathUri.toString
+      val filePath = new org.apache.hadoop.fs.Path(tablePath)
```

Comment:
I think you can directly create `Path` from `URI`.

Reply:
Done